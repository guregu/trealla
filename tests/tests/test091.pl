:-initialization(main).

main :-
	X1=f(L1),copy_term([123|X1],C1), writeln(C1),
	X2=f(L2),L2=[123|X2],copy_term(L2,C2), writeln(C2).
