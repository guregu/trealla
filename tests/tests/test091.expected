[123|f(_9)]
[123|f(_10)]
